### ISSUE TITLE
Vive controllers do not make chaperone appear in Echo Arena

### ISSUE THREAD

**Issue by MehStrongBadMeh** [LibreVR/Revive#652]
Something of an odd issue. When playing the Echo Arena Open beta, the chaperone only appears if my headset is near the boundaries. Moving my controllers near the boundaries does not cause the chaperone to appear, resulting in me repeatedly striking the wall. This does not happen while playing any SteamVR games nor any other Oculus games.

**Comment by jaorg1234** [comment]
Seems to be the same issue I had with Dead & Buried in the past here:

https://github.com/LibreVR/Revive/issues/445

Though CrossVR said Revive is not changing anything regarding the chaperone and it shouldn't influence its behavior.

One thing to check: did you by any chance press on "Reset Seated Position" in the SteamVR menu while staying in the middle of your play space?

**Comment by MehStrongBadMeh** [comment]
I did not use the "Reset Seated Position" feature as I had no problems with centering

**Comment by hockenmaier** [comment]
Yep, I can confirm this is happening to me too.  The chaperone appears but it seems there is a significant time delay.  I have actually had the chaperone appear nearly a second after I'd already collided with an object outside the bounds while playing echo arena.
